samples/Calling/src/app/App.tsx

```diff
@@ -102,8 +102,8 @@ const App = (): JSX.Element => {
   useEffect(() => {
     // Create a new CallClient when at the home page or at the createCallClient page.
     if (page === 'createCallClient' || page === 'home') {
-      const userIdFromToken = token ? getIdFromToken(token) : '';
-      const newStatefulCallClient = createStatefulCallClient({ userId: userIdFromToken });
+      // This sample app does not use `userId`. The stateful client internally never uses it either.
```

Comment:
There is also a usage of userId in calling-component-binding package. Is there any impact here with this change?

Reply:
Please see the latest commit in this PR which sets the userId to the correct value (just not one from `getIdFromToken`).

`userId` is indeed used from `ParticipantList` etc.